src/components/src/main/java/org/apache/jmeter/visualizers/ViewResultsFullVisualizer.java

```diff
@@ -135,9 +135,9 @@ public ViewResultsFullVisualizer() {
         super();
         final int maxResults = JMeterUtils.getPropDefault("view.results.tree.max_results", 500);
         if (maxResults > 0) {
-            buffer = new CircularFifoBuffer(maxResults);
+            buffer = new CircularFifoQueue<>(maxResults);
         } else {
-            buffer = new UnboundedFifoBuffer();
+            buffer = new LinkedList<>();
```

Comment:
Could we please stop using `LinkedList`?
Here's what Joshua Bloch (`LinkedList` author) says: https://twitter.com/joshbloch/status/583813919019573248

Reply:
What is your preferred List here?